notebooks/queries/q1.rq

```diff
@@ -0,0 +1,32 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX epo: <http://data.europa.eu/a4g/ontology#>
+PREFIX at: <http://publications.europa.eu/ontology/authority/>
+
+PREFIX locn: <http://www.w3.org/ns/locn#>
+PREFIX cv: <http://purl.org/captsolo/resume-rdf/0.2/cv#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX legal: <https://www.w3.org/ns/legal#>
+prefix dt:   <http://example.org/datatype#>
+prefix ns:   <http://example.org/ns#>
+prefix :     <http://example.org/ns#>
+prefix xsd:  <http://www.w3.org/2001/XMLSchema#>
+prefix foaf:  <http://xmlns.com/foaf/0.1/>
+
+SELECT ?BuyerCountry (count(distinct ?Lot) as ?Lots_using_EU_Funds) 
+{ 
+	?Contract rdf:type epo:Contract;
+            epo:includesLot ?Lot.
+  			?Lot rdf:type epo:Lot;
+            	epo:isUsingEUFunds true.
+    
+  
+  	?Contract epo:bindsBuyer ?Buyer.
+  			?Buyer rdf:type epo:Buyer;
+                   epo:playedBy ?Organization.
+  			?Organization rdf:type org:Organization;
+                          legal:registeredAddress ?Address.
+  			?Address rdf:type locn:Address;
+                     epo:hasCountryCode ?BuyerCountry.
+  
+
+} group by ?BuyerCountry
```

Comment:
I recommend to create a resource folder at level with notebooks, and put queries in resources/sparql_queries/*.rq .

Reply:
the resource folder was added and queries was moved there